packages/terra-site/src/examples/table/NoStripedTable.jsx

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import Table from 'terra-table';
+
+const NoStripedTable = () => (
+  <Table isStriped={false}>
+    <Table.Header>
+      <Table.HeaderContent content={'Column Heading 1'} key={1} minWidth={'small'} />
+      <Table.HeaderContent content={'Column Heading 2'} key={2} minWidth={'medium'} />
+      <Table.HeaderContent content={'Column Heading 3'} key={3} minWidth={'large'} />
+    </Table.Header>
+    <Table.Rows>
+      <Table.Row key={0}>
```

Comment:
We should leave it up to the user to set keys. Setting a random key is actually worse than just using the index since keys help React identify which items have changed, are added, or are removed.
Check this thread for some more information.
https://github.com/facebook/react/issues/1342